support-frontend/assets/components/subscriptionCheckouts/stripeForm/stripeForm.tsx

```diff
@@ -78,15 +83,23 @@ const marginTop = css`
 
 function StripeForm(props: StripeFormPropTypes): JSX.Element {
 	/**
-	 * State
+	 * Redux hooks
+	 */
+	const dispatch = useSubscriptionsDispatch();
+	const setupIntentClientSecret = useSubscriptionsSelector(
+		(state) => state.page.checkoutForm.payment.stripe.setupIntentClientSecret,
+	);
+	const recaptchaCompleted = useSubscriptionsSelector(
+		(state) => state.page.checkoutForm.recaptcha.completed,
+	);
```

Comment:
Using multiple selectors is [recommended](https://redux.js.org/style-guide/#call-useselector-multiple-times-in-function-components) to minimise potential re-renders.